### ISSUE TITLE
HTML entities appear escaped in browser title bar

### ISSUE THREAD

**Issue by gwagener** [refinery/refinerycms#1242]
HTML entities that are passed into the page title via `content_for :title` are displayed in the browser title as their escape string, e.g. `&lt;` for `<`.  This occurs because the entities are automatically escaped and then converted to a string in the `browser_title` helper method, then the value of that method is automatically escaped and unescaped.

**Comment by evenreven** [comment]
@parndt Sorry for bumping a 10 years old issue, but I noticed you merged #1243 into the 1.0-stable branch. However, master and 4.0-stable does not have this commit. Is there a reason for that? Is it unsafe to use html_safe here? 

I've had this issue since 2015, but I've managed to mostly work around it by using proper quotes («») in titles whenever I needed them, but [now I have a page](https://khio.no/events/1497) with an ampersand in the title (which shows up as `Give Them Wings &amp; We Shall See Their Faces - Performance-in-Progress - Kunsthøgskolen i Oslo` in the `head`), so I got interested in solving it properly. Since it seems to be escaped twice (the actual html is `<meta name="title" content="Give Them Wings &amp;amp; We Shall See Their Faces - Performance-in-Progress - Kunsthøgskolen i Oslo">`), I'd guess html_safe is safe enough here? 

If so, I'll just put it in a decorator in my app (which I'm using anyway because of some custom models), but I figured I should ask whether it's safe since you've done this change once already. :)

(I tried it and it didn't break, but maybe I'm overlooking something.)
